cpp/ql/test/library-tests/dataflow/fields/struct_init.c

```diff
@@ -12,7 +12,7 @@ struct Outer {
 };
 
 void absink(struct AB *ab) {
-  sink(ab->a); // flow x3 [NOT DETECTED]
+  sink(ab->a); // flow (three paths)
```

Comment:
This would be better if it were three separate sources. Losing a path would be easier to miss than losing a source.

Reply:
Done.